### PR TITLE
 refactor(components, app): move LargeButton to components and add helix styling

### PR DESCRIPTION
**Pull request by jerader**
closes AUTH-622

# Overview

move `LargeButton` to components for the PD component work.

## Test Plan and Hands on Testing

review code and review storybook. For the storybook, make sure to manually change the viewport size to see helix vs odd

## Changelog

move `LargeButton` to components and update all associated imports. add styling for helix

## Review requests

see test plan

## Risk assessment

low

### CODE REVIEW

File: components/src/atoms/buttons/LargeButton.tsx [hunk 1]
```diff
@@ -126,80 +129,103 @@ export function LargeButton(props: LargeButtonProps): JSX.Element {
...
-    ${TYPOGRAPHY.pSemiBold}
-
-    #btn-icon: {
-      color: ${disabled
-        ? LARGE_BUTTON_PROPS_BY_TYPE[buttonType].disabledIconColor
-        : LARGE_BUTTON_PROPS_BY_TYPE[buttonType].iconColor};
-    }
+    padding: ${SPACING.spacing16} ${SPACING.spacing24};
+    text-align: ${TYPOGRAPHY.textAlignCenter};
+    border-radius: ${BORDERS.borderRadius40};
```
Comment (koji) [inline components/src/atoms/buttons/LargeButton.tsx]: https://www.figma.com/design/WbkiUyU8VhtKz0JSuIFA45/Feature%3A-Protocol-Designer-Phase-1?node-id=5476-13797&m=dev

the design is using 200px but I think 40px would be enough.

File: components/src/atoms/buttons/LargeButton.tsx [hunk 2]
```diff
@@ -126,80 +129,103 @@ export function LargeButton(props: LargeButtonProps): JSX.Element {
...
+    flex-direction: ${DIRECTION_ROW};
```
Comment (koji) [inline components/src/atoms/buttons/LargeButton.tsx]: need this?

File: components/src/atoms/buttons/LargeButton.tsx (hunk 1 above)
Comment (jerader) [reply components/src/atoms/buttons/LargeButton.tsx]: thanks, good catch!

File: components/src/atoms/buttons/LargeButton.tsx (hunk 2 above)
Comment (jerader) [reply components/src/atoms/buttons/LargeButton.tsx]: oh good call, i always forget that flex direction row is implicit so this is not needed

**Review by koji — APPROVED** [review]
lgtm

**Pull request merged**